tests/testthat/transformer_grouping/scope_none-out.R

```diff
@@ -0,0 +1,30 @@
+#'this function does
+#'
+#' @param x a parameter.
+#'    indented comments
+a<- function(x){
+  test_that("I want to test",{
+    out <- c(1,c(
+      22 +1
+    ))
+    if (x > 10) {
+      for (x in 22) { # FIXME in operator only to be surrounded by one space. What about %in%?
+        prin(x)
+      }
+    }
+  }     )
+  #we like comments too
+  c(list(x + 2),
+    c(    c(
+      26 ^ 2, # FIXME ^ operator has to be surrounded by one space (or none?!), never multiple
```

Comment:
I think a space is fine, but we could offer alternatives (e.g., space only around + and -, never around *, /, ^).